mlflow/ml-package-versions.yml

```diff
@@ -203,7 +203,7 @@ fastai-1.x:
     pip_release: "fastai"
 
   models:
-    minimum: "1.0.60"
+    minimum: "1.0.30"
```

Comment:
Sorry I should have clarified. We don't need to change the minimum version for `fastai`.